### ISSUE TITLE
Edit Dialog: pop-up too large

### ISSUE THREAD

**Issue by llemeurfr** [toggl/track-macos-feedback#61]
## Summary

- [ ] Crash 
- [v] Glitch 
- [ ] Sync 

When editing a task from the List of Calendar, the pop-up gets too large. This is the case whatever screen I use (mono or dual screen). 

Note: otherwise the app is great!

![Capture d’écran 2023-07-03 à 11 44 33](https://github.com/toggl/track-macos-feedback/assets/14943186/6f19643c-c004-423f-a8b3-0baf9d4acc41)

## Affected versions

**App version: 8.11.1** 
**OS version: MacOS**
**Device model: Macbook Pro**

**Comment by certainlyakey** [comment]
This is a likely duplicate of the issue i've been experiencing (#57).

**Comment by AndrewVebster** [comment]
Just wanted to let you know that this issue should be fixed in the latest 9.0.1 release.

I'm closing the issue but if it's not fixed for you, feel free to reopen it again.

**Comment by certainlyakey** [comment]
Yes, you need to empty the container folder. Sent from my iPhoneOn 3. Oct 2023, at 16:35, L. Le Meur ***@***.***> wrote:﻿
No, sorry it didn't solve the problem for me. I deleted the app (on Mac), emptied the bin, downloaded the latest version (9.1.0) and the edit screen still takes almost the screen size (I've got 2 screens, the width adapts to the screen).
Is there a hidden file containing some parameters I should delete?
